components/ResourceHub/company-resources.json

```diff
@@ -382,6 +62,20 @@
       "Safe Smart Account",
       "7579"
     ],
-    "image": "https://i.ytimg.com/vi/d-wGezfH6nM/maxresdefault.jpg?sqp=-oaymwEmCIAKENAF8quKqQMa8AEB-AH-CYAC0AWKAgwIABABGE8gUShlMA8=&rs=AOn4CLCENcg7N7HKxVrN2crekt04CWlkvg"
+    "image": "https://i.ytimg.com/vi/d-wGezfH6nM/maxresdefault.jpg?sqp=-oaymwEmCIAKENAF8quKqQMa8AEB-AH-CYAC0AWKAgwIABABGE8gUShlMA8=&rs=AOn4CLCENcg7N7HKxVrN2crekt04CWlkvg",
+    "lastChecked": "2024-08-09"
+  },
+  {
+    "name": "Universal Wallets with Safe",
+    "url": "https://www.youtube.com/watch?v=h40Gy3fsRJg",
+    "type": "Video",
+    "date": "2024-08-08",
+    "description": "A workshop on Universal Wallets with Safe.",
```

Comment:
Should we mention the AppKit as well? As Safe is integrated inside WalletConnect's AppKit